packages/api-client/src/replayArchivedS3Messages.ts

```diff
@@ -0,0 +1,35 @@
+import { invokeApi } from './cumulusApiClient';
+import { ApiGatewayLambdaHttpProxyResponse, InvokeApiFunction } from './types';
+
+/**
+ * POST a request to start replaying archived S3 Messages
+ *
+ * @param {Object} params              - params
+ * @param {string} params.prefix       - the prefix configured for the stack
+ * @param {string} params.queueName    - name of queue to queue archived messages
+ * @param {Function} params.callback   - async function to invoke the api lambda
+ *                                       that takes a prefix / user payload.  Defaults
+ *                                       to cumulusApiClient.invokeApi
+ * @returns {Promise<Object>}          - promise that resolves to the output
+ *                                       of the API lambda
+ */
+export const postReplayArchivedMessages = async (params: {
```

Comment:
```suggestion
export const replaySqsMessages = async (params: {
```

I would also move it to the `replays` file in here depending on if we end up moving the endpoint